### PR TITLE
[#35] Add prettier

### LINKED ISSUE #35: Add Prettier for automatic code formatting
Replace `eslint-config-airbnb` with `eslint:recommended` preset, add Prettier and run it automatically before commit.

### PR DESCRIPTION
**Pull request by alik0211**
Resolve #35

### CODE REVIEW

File: .eslintrc [hunk 1]
```diff
@@ -22,15 +33,17 @@
     "no-multiple-empty-lines": ["error", {
       "max": 1
     }],
-    "import/no-extraneous-dependencies": ["error", {
-      "devDependencies": ["**/*.story.js", "**/webpack/**/*.js"]
-    }],
+    "import/no-extraneous-dependencies": "off",
```
Comment (andrew--r) [inline .eslintrc]: Why is it changed?

File: .prettierrc [hunk 2]
```diff
@@ -0,0 +1,10 @@
+{
+  "useTabs": true,
+  "printWidth": 80,
+  "tabWidth": 2,
+  "singleQuote": true,
+  "trailingComma": "es5",
+  "jsxBracketSameLine": false,
+  "parser": "flow",
+  "semi": true
+}
```
Comment (andrew--r) [inline .prettierrc]: There is no new line in the end of the file. Do you have [editorconfig](https://editorconfig.org/) plugin enabled in your IDE/editor? It should have inserted new line automatically.

File: package.json [hunk 3]
```diff
@@ -34,28 +37,31 @@
...
+    "babel-preset-react": "6.24.1",
     "cross-env": "5.1.5",
     "css-loader": "0.28.11",
-    "eslint": "5.5.0",
-    "eslint-config-airbnb": "^10.0.1",
-    "eslint-plugin-import": "^1.13.0",
-    "eslint-plugin-jsx-a11y": "^2.1.0",
-    "eslint-plugin-react": "^6.1.1",
+    "eslint": "5.6.0",
+    "eslint-config-prettier": "^3.3.0",
```
Comment (andrew--r) [inline package.json]: Could you please set exact versions for `eslint-config-prettier` and `eslint-plugin-prettier`?

File: package.json [hunk 4]
```diff
@@ -34,28 +37,31 @@
...
+    "eslint-plugin-react": "7.11.1",
     "html-webpack-plugin": "3.2.0",
     "husky": "0.14.3",
     "jest": "22.4.3",
     "lint-staged": "7.1.0",
     "mini-css-extract-plugin": "^0.4.0",
     "postcss-assets": "5.0.0",
     "postcss-loader": "2.1.5",
+    "prettier": "1.14.3",
+    "prop-types": "15.6.2",
```
Comment (andrew--r) [inline package.json]: Why `prop-types` is added as a dev dependency? It is used as a production dependency.

File: .prettierrc (hunk 2 above)
Comment (alik0211) [reply .prettierrc]: Fix - 0401711e811867528918a532b1e15fec48ba7cde

**Comment by andrew--r** [comment]
Thanks!

**Pull request merged**